### PR TITLE
fix(trashbin): export config

### PR DESCRIPTION
**Pull request by eye-wave**
I got `NotAllowedError: Write permission denied.` when exporting, so i added a catch block with deprecated execCommand copy

### CODE REVIEW

File: Extensions/trashbin.js [hunk 1]
```diff
@@ -303,8 +303,19 @@
...
+        const tempElement =document.createElement("textarea");
+        tempElement.value =JSON.stringify(data);
+        
+        document.body.append(tempElement);
+        tempElement.select();
+        document.execCommand("copy");
+        
+        tempElement.remove();
+      })
+      .finally(() => Spicetify.showNotification("Copied to clipboard"));
```
Comment (kyrie25) [inline Extensions/trashbin.js:318]:
```suggestion
		await Spicetify.Platform.ClipboardAPI.copy(JSON.stringify(data));
		Spicetify.showNotification("Copied to clipboard"));
```

**Review by kyrie25 — CHANGES_REQUESTED** [review]

**Comment by rxri** [comment]
@eye-wave, why do you mark @kyrie25 suggestion as resolved, when it's not?

**Comment by eye-wave** [comment]
It's not?
Sorry, didn't mean to make your work harder. I'm still learning about all of this

**Comment by rxri** [comment]
You didn't commit his suggestion, so conversation isn't resolved yet

**Comment by ohitstom** [comment]
i forgot spicetify had its own clipboard api, i assumed it worked the same way though... my bad...

File: Extensions/trashbin.js (hunk 1 above)
Comment (eye-wave) [reply Extensions/trashbin.js:318]: I didn't know about that Spicetify.Platform.ClipboardAPI.
you forgot to make the function async tho to use await

File: Extensions/trashbin.js [hunk 2]
```diff
@@ -303,8 +303,8 @@
 			songs: trashSongList,
 			artists: trashArtistList
 		};
-		navigator.clipboard.writeText(JSON.stringify(data));
-		Spicetify.showNotification("Copied to clipboard");
+		await Spicetify.Platform.ClipboardAPI.copy(JSON.stringify(data));
+		Spicetify.showNotification("Copied to clipboard"));
```
Comment (rxri) [inline Extensions/trashbin.js:307]:
```suggestion
		Spicetify.showNotification("Copied to clipboard");
```

**Review by rxri — CHANGES_REQUESTED** [review]

File: Extensions/trashbin.js (hunk 1 above)
Comment (kyrie25) [reply Extensions/trashbin.js]: > I didn't know about that Spicetify.Platform.ClipboardAPI.
> 
> you forgot to make the function async tho to use await

The API method is asynchronous, also it doesn't necessarily have to be asynchronous function to use await.

Comment (kyrie25) [reply Extensions/trashbin.js]: > I didn't know about that Spicetify.Platform.ClipboardAPI. you forgot to make the function async tho to use await

It was a suggestion of an alternative method, not that I completely fixed it.

File: Extensions/trashbin.js [hunk 3]
```diff
@@ -303,7 +303,7 @@
 			songs: trashSongList,
 			artists: trashArtistList
 		};
-		navigator.clipboard.writeText(JSON.stringify(data));
+		await Spicetify.Platform.ClipboardAPI.copy(JSON.stringify(data));
```
Comment (kyrie25) [inline Extensions/trashbin.js:306]: Either make the function asynchronous or do this
```suggestion
		Spicetify.Platform.ClipboardAPI.copy(JSON.stringify(data));
```

**Review by kyrie25 — CHANGES_REQUESTED** [review]

**Review by kyrie25 — APPROVED** [review]

**Review by rxri — APPROVED** [review]

**Pull request merged**